### PR TITLE
♻️ Refactors webserver module ``director_v2`` and breaks dependency cycles

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

This PR refactors ``web-server`` service to prepare for new features that will be introduce in a separate PR #2556

- ♻️ Splits ``director-v2`` app module into submodules for 
   - *_settings*: Defines ``BaseSettings``-based settings with options to configure this module
   - *_handlers*: defines handlers registered in the ``aiohttp``'s ``app.router``
   - *_core*: includes core functionality
   - *_api*: exposes functions to other app modules
- ♻️ Splits ``project`` app module's handler into smaller submodules
- 🐛 Replaces built-in ``TimeoutError`` by ``asyncio.TimeoutError``. Those are differemt things.
- ♻️ Cleanup ``simcore_service_webserver.utils``
- ♻️ Removed dependency cycles. 
```cmd
$ cat $HOME/.pydeps
[pydeps]
max_bacon = 3
no_show = True
verbose = 1
pylib = False
exclude =
    os
    re
    sys
    collections
    __future__
    aiohttp
    __main__
    celery
    attr
    yaml
$ pip install pydeps   
$ pydeps src/simcore_service_webserver --show-cycles
```
image

- 

## Related issue/s
- #2392 
  - Refactors directorv2 app-module to introduce PR #2556

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/version_control_db.py [hunk 1]
```diff
@@ -40,6 +42,10 @@
 
 
 def compute_checksum(workbench: Dict[str, Any]) -> SHA1Str:
+    #
+    # - UI is NOT accounted in the checksum
+    # - TODO: review other fields to mask?
+    #
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/version_control_db.py:48]: Could you clear out this a bit more?

**Review by GitHK — APPROVED** [review]
very nice.

**Review by sanderegg — APPROVED** [review]
ok... I'm a little bit scared of my next rebase ;)

File: services/web/server/src/simcore_service_webserver/version_control_db.py (hunk 1 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/version_control_db.py:48]: Aha sorry, this got here while it was a comment intended for PR #2556. I will remove it from here.

**Pull request merged**